### PR TITLE
default pad brightness : 4-100 value from UI with 4-step offset

### PR DESCRIPTION
**Pull request by joxihan**
not sure if it counts as a bug fix, but based on feedback i got from people using it on nightly, might be a good addition to 1.1 considering the low risk

### CODE REVIEW

File: src/deluge/gui/menu_item/defaults/pad_brightness.h [hunk 1]
```diff
@@ -23,12 +23,21 @@ namespace deluge::gui::menu_item::defaults {
 class PadBrightness final : public Integer {
 public:
 	using Integer::Integer;
-	[[nodiscard]] int32_t getMinValue() const override { return kMinLedBrightness; }
-	[[nodiscard]] int32_t getMaxValue() const override { return kMaxLedBrightness; }
-	void readCurrentValue() override { this->setValue(FlashStorage::defaultPadBrightness); }
+	void selectEncoderAction(int32_t offset) override {
+		offset = offset * 4;
```
Comment (m-m-adams) [inline src/deluge/gui/menu_item/defaults/pad_brightness.h:27]: Offset is already multiplied by 5 if you're holding shift - this shouldn't be needed

Comment (joxihan) [reply src/deluge/gui/menu_item/defaults/pad_brightness.h:27]: OK maybe I can add this in the doc as testers were complaining about it and removing the *4 offset

Comment (sapphire-arches) [reply src/deluge/gui/menu_item/defaults/pad_brightness.h:27]: this *4 is serving a different purpose, preventing the selection of brightness values that don't map to a valid interval setting. I think it's ok as is.

Comment (m-m-adams) [reply src/deluge/gui/menu_item/defaults/pad_brightness.h:27]: Right, ok that makes sense

**Review by m-m-adams — APPROVED** [review]

**Pull request merged**